wca-regulations.md

```diff
@@ -211,15 +211,21 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 - 9a) The WCA governs competitions for mechanical puzzles that are operated by twisting groups of pieces, commonly known as "twisty puzzles".
 - 9b) The official events of the WCA are:
-    - 9b1) 3x3x3 Cube, 2x2x2 Cube, 4x4x4 Cube, 5x5x5 Cube, 3x3x3 One-Handed, 3x3x3 With Feet, Clock, Megaminx, Pyraminx, Skewb, and Square-1.
-        - 9b1a) Round format for these events is: "Average of 5". For a Combined Round, the round format is "Best of 2" for the first phase and "Average of 5" for the second phase.
-    - 9b2) 6x6x6 Cube and 7x7x7 Cube.
-        - 9b2a) Round format for these events is: "Mean of 3". For a Combined Round, the round format is "Best of 1" for the first phase and "Mean of 3" for the second phase.
-    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded, and 3x3x3 Multi-Blind.
-        - 9b3a) Round formats for these events are: "Best of X" (where X is 1, 2, or 3).
-        - 9b3b) For 3x3x3 Blindfolded, 4x4x4 Blindfolded, and 5x5x5 Blindfolded the WCA also recognizes "Mean of 3" rankings and records based on the times from "Best of 3" rounds.
+    - 9b1) 3x3x3 Cube, 2x2x2 Cube, 4x4x4 Cube, 5x5x5 Cube, 3x3x3 One-Handed, 3x3x3 With Feet, Clock, Megaminx, Pyraminx, Skewb, and Square-1:
+        - 9b1a) Round format for these events: "Average of 5".
+        - 9b1b) Combined Round format for these events: "Best of 2" cutoff phase for "Average of 5".
+    - 9b2) 6x6x6 Cube and 7x7x7 Cube, with the following round formats:
+        - 9b2a) Round format for these events: "Mean of 3".
+        - 9b2b) Combined Round format for these events: "Best of 1" cutoff phase for "Mean of 3".
+    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded.
+        - 9b3a) Round format for these events: "Best of 3".
+        - 9b3b) For these events, the WCA also recognizes "Mean of 3" rankings and records based on the times from "Best of 3" rounds.
     - 9b4) 3x3x3 Fewest Moves.
-        - 9b4a) Round formats for this event are: "Best of X" (where X is 1 or 2) and "Mean of 3".
+        - 9b4a) Round formats for this event: Best of X" (where X is 1 or 2) or "Mean of 3".
```

Comment:
Is there any reason for the "and" to "or" change?
(the suggestion also includes a missing `"`)
```suggestion
        - 9b4a) Round formats for this event: "Best of X" (where X is 1 or 2) and "Mean of 3".
```

Reply:
I want to avoid the chance of giving the impression that `"Best of 2" and "Mean of 3"` looks like it describes a combined round. I think "or" works just as well, and is more likely to be understood correctly without thinking about it.

Reply:
```suggestion
        - 9b4a) Round formats for this event: "Best of X" (where X is 1 or 2) or "Mean of 3".
```

Reply:
I was actually just wondering about the reasoning behind the change, I definitely don't feel strongly so "or" is ok!
I think you can go ahead with your last suggested change!